lib/parser.dart

```diff
@@ -1213,7 +1215,7 @@ class _Parser {
       Declaration decl = processDeclaration(dartStyles);
       if (decl != null) {
         if (decl.hasDartStyle) {
-          var newDartStyle = decl.dartStyle;
+          var newDartStyle = decl.dartStyle as DartStyleExpression;
```

Comment:
Any reason `Declaration.dartStyle` couldn't just be typed as `DartStyleExpression`?

Reply:
Good catch; looks like it can (constructor is only called in two places). Done.